src/app/page.tsx

```diff
@@ -14,300 +12,306 @@ export default function Home() {
-                  <td className="p-4">
-                    <button type="button" className="primary small" disabled>
-                      Disabled
-                    </button>
-                  </td>
-                </tr>
-                <tr>
-                  <td className="p-4 font-medium bg-gray-100">secondary small</td>
-                  <td className="p-4">
-                    <button type="button" className="secondary small">
-                      Secondary
-                    </button>
-                  </td>
-                  <td className="p-4">
-                    <button type="button" className="secondary small" disabled>
-                      Disabled
-                    </button>
-                  </td>
-                </tr>
-                <tr>
-                  <td className="p-4 font-medium bg-gray-100">tertiary small</td>
-                  <td className="p-4">
-                    <button type="button" className="tertiary small">
-                      Tertiary
-                    </button>
-                  </td>
-                  <td className="p-4">
-                    <button type="button" className="tertiary small" disabled>
-                      Disabled
-                    </button>
-                  </td>
-                </tr>
-                <tr>
-                  <td className="p-4 font-medium bg-gray-100">primary danger</td>
-                  <td className="p-4">
-                    <button type="button" className="primary danger">
-                      Primary
-                    </button>
-                  </td>
-                  <td className="p-4">
-                    <button type="button" className="primary danger" disabled>
-                      Disabled
-                    </button>
-                  </td>
-                </tr>
-                <tr>
-                  <td className="p-4 font-medium bg-gray-100">secondary danger</td>
-                  <td className="p-4">
-                    <button type="button" className="secondary danger">
-                      Secondary
-                    </button>
-                  </td>
-                  <td className="p-4">
-                    <button type="button" className="secondary danger" disabled>
-                      Disabled
-                    </button>
-                  </td>
-                </tr>
-                <tr>
-                  <td className="p-4 font-medium bg-gray-100">tertiary danger</td>
-                  <td className="p-4">
-                    <button type="button" className="tertiary danger">
-                      Tertiary
-                    </button>
-                  </td>
-                  <td className="p-4">
-                    <button type="button" className="tertiary danger" disabled>
-                      Disabled
-                    </button>
-                  </td>
-                </tr>
-              </tbody>
-            </table>
+    */ <>
+      <Header />
```

Comment:
The header should go on the layout.tsx page so it applies to all pages

Reply:
layout page is SBCQ-65, which is not done yet